src/components/Roadmap.js

```diff
@@ -57,6 +57,18 @@ const Roadmap = () => {
     implemented: blankIssues,
   })
 
+  const checkifMerge= async (url)=>{
+    let resp = await axios.get(url+"merge")
+    if(resp.status===203){
+      
+      return true
+      
+    }
+    else{
+      return false
+    }
+  }
+
```

Comment:
Cool, will check this out when it builds...
Keep in mind `resp.status === 203` is a boolean statement by itself, and you can just return that, ie:
```
const resp = await axios.get(url + "merge")
return resp.status === 203 // This will return true or false depending on output
```

Reply:
@wackerow I am not able to check locally. It shows Loading error Please refersh at localhost:8000. I have added github API key and etherscan key still its not working. Can you pls guide me

Reply:
Ok Sure I will change it